deps/config/migrations/000001_create_tables.down.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
```

Comment:
Migrations are not config, therefore I would rather place them un level un, just in `deps/migration` or even at the root `/migrations`

Reply:
done